component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -29,6 +29,7 @@
     public static final int SECRET_KEY_LENGTH = 5;
     public static final int NUMBER_BASE = 2;
     public static final int NUMBER_DIGIT = 6;
+    public static final String EMPTY_STRUNG = "";
```

Comment:
Can we use `StringUtils.EMPTY`? I think it's better than defining an empty constant. WDYT?